Presentation.Web/app/components/home/home-edit-user.controller.ts

```diff
@@ -65,6 +69,18 @@
 
             //can't use autosave - need to patch through userService!
             $scope.patch = function (field, value) {
+
+                if (value === undefined) {
+                    if (field === $scope.nameFieldName ||
+                        field === $scope.lastNameFieldName ||
+                        field === $scope.emailFieldName) {
+                        return;
+                    }
+                }
+                if (field === 'name' && value === undefined) {
```

Comment:
why not use the `$scope.nameFieldName` as in the condition above?
And why check it twice.. should already be included in the previous check right?